commissions/tests.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
+from .models import Commission, Comment
+
+class CommissionModelTest(TestCase):
+    def setUp(self):
+        self.commission = Commission.objects.create(
+            title="Sample Commission",
+            description="This is a test commission.",
+            people_required=3
+        )

```

Comment:
Please call `self.commission.save()`. I'm not sure if it's okay to use `self.commission` instead of just `commission`? I'm not too knowledgeable on this and will consult with sir.